framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/GenericScenarioImpl.java

```diff
@@ -170,9 +186,54 @@ protected void deployKieDeployments() {
             kieServerDeployment.waitForScale();
         }
 
+        // check if there is an AMQ deployment
+        try {
+            logger.info("Searching for AMQ.");
+            ServiceUtil.getAmqJolokiaServiceName(project.getOpenShift());
```

Comment:
What is this used for?

Reply:
This search for an AMQ in generic scenario. If AMQ was created by generic scenario this code create an AmqDeployment so in test can be get data from deployments (e.g. url).